src/vue/forms/RedeemForm.vue

```diff
@@ -158,6 +166,12 @@ export default {
       const n = Math.random() * Number(MAX_INT_32)
       return n.toFixed()
     },
+
+    setDefaultAmount () {
+      this.form.amount = this.accountBalance.balance >= 1
```

Comment:
Are you sure that `this.accountBalance.balance` will be a number ?

Reply:
made a number